crates/uv-cli/src/lib.rs

```diff
@@ -291,6 +291,20 @@ pub struct CleanArgs {
     pub package: Vec<PackageName>,
 }
 
+#[derive(Args, Debug)]
+#[allow(clippy::struct_excessive_bools)]
+pub struct PruneArgs {
+    /// Whether to remove unzipped wheels from the cache, leaving only zipped wheel entries.
+    ///
+    /// By default, uv stores unzipped wheels in the cache, which enables high-performance package
+    /// installation. In some scenarios, though, persisting unzipped wheels may be undesirable. For
+    /// example, in GitHub Actions or other CI environments, uploading unzipped wheels to a remote
+    /// cache may have a negative impact on cache performance. Pruning unzipped wheels will leave
+    /// the cache with any built wheels in their zipped form.
+    #[arg(long)]
+    pub all_unzipped: bool,
```

Comment:
What about calling it `--ci` and documenting it as keeping only the artifacts you want to upload to the CI cache?

Reply:
From a user perspective, I do feel this would be the main use-case. It would also make it immediately clear to me that I probably should use this in CI pipelines.